samples/msgext-doc-compliance-checker/env/.env.dev

```diff
@@ -0,0 +1,16 @@
+# This file includes environment variables that will be committed to git by default.
+
+# Built-in environment variables
+TEAMSFX_ENV=dev
+APP_NAME_SUFFIX=dev
+
+# Updating AZURE_SUBSCRIPTION_ID or AZURE_RESOURCE_GROUP_NAME after provision may also require an update to RESOURCE_SUFFIX, because some services require a globally unique name across subscriptions/resource groups.
+AZURE_SUBSCRIPTION_ID=
+AZURE_RESOURCE_GROUP_NAME=
+RESOURCE_SUFFIX=
+
+# Generated during provision, you can also add your own variables.
+BOT_ID=
```

Comment:
I can't see blob storage connect or Azure Open AI service related keys here.

Reply:
As I understand it, there is no need to update the local environment file; we only need to update the .**localConfigs** file.